include/LightGBM/dataset.h

```diff
@@ -739,6 +754,36 @@ class Dataset {
     return raw_data_[numeric_feature_map_[feat_ind]].data();
   }
 
+  inline void store_mapping()  {
+    // store maps to mapping
+    int offset = 0;
+    for(int i=0; i < maps.size(); i++){    
+      std::string name;
+      
+      if(i == label_idx_){
```

Comment:
> if( [](http://example.com/codeflow?start=5&length=4)

same one, please review the other code to make sure you are following the code style